src/main/java/com/google/firebase/auth/FirebaseUserManager.java

```diff
@@ -226,6 +228,16 @@ UserImportResult importUsers(UserImportRequest request) throws FirebaseAuthExcep
     return new UserImportResult(request.getUsersCount(), response);
   }
 
+  Tenant getTenantById(String tenantId) throws FirebaseAuthException {
```

Comment:
I think `getTenant` is descriptive enough. We have several different ways of fetching user accounts (by UID, by email etc), but we don't have that requirement here.

Reply:
Done.